lms/static/js/ccx/collection/schedule_collection.js

```diff
@@ -0,0 +1,269 @@
+var define = window.define || RequireJS.define;  // jshint ignore:line
```

Comment:
I would recommend that you follow our standard practice of passing define in as an argument of an executed anonymous function. For example, see https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/views/AccessView.js